src/productListScraper.js

```diff
@@ -54,14 +54,15 @@ var findProducts = function () {
   Gets the next page on a vinbud product search page. Depends on a PhantomJS DOM.
  */
 var getNextPage = function () {
-  var next = document.querySelector('span.selectedpage').nextSibling;
+  var current = document.querySelector('.selectedpage');
+  var next = document.querySelector('.paging_button_next');
 
-  if (!next || (next && !next.classList.contains('notselectedpage'))) {
+  if (!next || (next && next.classList && next.classList.contains('aspNetDisabled'))) {
```

Comment:
I switched up the pagination a bit to be more clear on what was happening in code.